projects/Mallard/src/App.tsx

```diff
@@ -82,6 +86,21 @@ const shouldHavePushFailsafe = async (client: ApolloClient<object>) => {
 	}
 };
 
+const forceUpdateApiUrlIfBeta = async () => {
+	// TODO - to test our new edition stack we are forcing beta users to use
```

Comment:
Do we need to update/remove this comment in light of the change in logic?

Reply:
Thanks @groakland . The actual comments is probably not very useful but TODO is just to remind us that this logic can be deleted once we fully migrate. I will update the comment